lib/order.rb

```diff
@@ -0,0 +1,89 @@
+require_relative "customer"
+
+class Order
+  attr_reader :id
+  attr_accessor :products, :customer, :fulfillment_status
+
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id
+    @products = products #hash
+    @customer = customer #instance of Customer
+    valid_statuses = [:pending, :paid, :processing, :shipped, :complete]
+    if !valid_statuses.include? fulfillment_status
+      raise ArgumentError.new("Invalid status")
+    end
+    @fulfillment_status = fulfillment_status
+  end
+
+  def total()
+    sum = 0
+    @products.each do |product, cost|
+      sum += cost
+    end
+
+    sum_with_tax = (sum + (sum * 0.075)).round(2)
+
+    return sum_with_tax
```

Comment:
You don't have to assign to a variable before returning.

```suggestion
    return (sum + (sum * 0.075)).round(2)
```